### DISCUSSION TITLE
Voteparty Feature

### DISCUSSION THREAD

**Discussion by mario414x** [Zrips/CMI#8173] [Feature Requests]
Hello! What do you think about a vote party feature? It would execute some commands when a certain number of votes is reached on the entire server. Also, if would be great if we can have specialised commands support.

**Comment by CloudeLecaw** [comment]
duplicated request. pls close it. and may add yoir request to one of the allready existing once.

(and NO it doesnt matter if they are 2 or 4 years old if they are still opend)

https://github.com/Zrips/CMI/discussions/3202
https://github.com/Zrips/CMI/discussions/5204

![Screenshot_2023-10-15-15-38-40-568_com android chrome-edit](https://github.com/Zrips/CMI/assets/12080017/9f2d15d6-a78b-4be3-bbb3-a56cfe3f5ad4)
